.gitlab-ci.yml

```diff
@@ -172,8 +174,21 @@ test-observability-pipelines-worker:
       - IMAGE: rockylinux:9.0
       - IMAGE: ubuntu:20.04
       - IMAGE: ubuntu:22.04
+      - IMAGE: amazonlinux:2
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ Missed the additional variable in this matrix variant.
```suggestion
      - IMAGE: amazonlinux:2
        INSTALL_CLASSIC_AGENT: --opw-install-classic-agent install_script_agent7.sh
```